### PR TITLE
allow scss variable overloading

### PR DESCRIPTION
**Pull request by ebrehault**

### CODE REVIEW

File: projects/pastanaga/src/lib/button/button.component.scss [hunk 1]
```diff
@@ -18,45 +18,35 @@ pa-button {
     vertical-align: middle;
     border-radius: 10rem;
     font-family: $font-family-base;
-    text-transform: uppercase;
+    text-transform: $labels-text-transform;
```
Comment (mathilde-pellerin) [inline projects/pastanaga/src/lib/button/button.component.scss]: might be a bit too generic: are we sure all the labels through pastanaga will have the same text transform?

File: projects/pastanaga/src/lib/button/button.component.scss [hunk 2]
```diff
@@ -134,6 +124,15 @@ pa-button {
             color: $red-dark;
         }
     }
+    &#{&}-contrast {
+        color: $white;
```
Comment (mathilde-pellerin) [inline projects/pastanaga/src/lib/button/button.component.scss]: should be `$button-contrast-color` (same for colors below, use the proper variables)

File: src/styles.scss [hunk 3]
```diff
@@ -1,5 +1,12 @@
 $font-path: '../projects/pastanaga/src/assets/fonts';
 @import "../projects/pastanaga/src/lib/styles/common-reset";
+@import "../projects/pastanaga/src/lib/styles/fonts";
+
+// for custom theming, set CSS variables like this:
```
Comment (mathilde-pellerin) [inline src/styles.scss]: would be good to have this example in the readme as well

Comment (ebrehault) [reply src/styles.scss]: it is already in the README

Comment (mathilde-pellerin) [reply src/styles.scss]: well, not exactly but good enough :wink:

**Pull request merged**